frontend/src/components/Prize/index.js

```diff
@@ -5,6 +5,7 @@ import Text from "components/Typography/Text";
 import Title from "components/Typography/Title";
 
 import styles from "./index.module.css";
+import { COMPANY } from "constants/Subvisual.js";
 
 // Prizes
 import Tshit from "assets/imgs/prizes/tshirt.png";
```

Comment:
Fix spelling.

Reply:
Same situation. Is fixed in another PR but I'll fix anyways

Reply:
From Slack:

Added my final review messages for PR#35 - Let’s get it all fixed and merged! (We will be closing PR#28 & PR#29 without merging so make sure to include all changes in PR#35)